quickstack/manifests/neutron/compute.pp

```diff
@@ -188,7 +188,7 @@
     $local_ip = find_ip("$ovs_tunnel_network","$ovs_tunnel_iface","")
     class { '::neutron::agents::ml2::ovs':
       bridge_uplinks   => $ovs_bridge_uplinks,
-      bridge_mappings  => $ovs_bridge_mappings,
+      #bridge_mappings  => $ovs_bridge_mappings,
```

Comment:
Initially, I did the change of adding another VLAN tagged interface and using that for the communication between controller and compute. The disadvantage of this was that the ip-address of this new interface was then getting assigned to br-ex (both controller and compute) and floating ip-address won't work in this case since floating-ip's needs to be public address range and br-ex is now configured with private ip-address.

To overcome this issue, I found a much simpler alternative way.
Use the same configuration, just change the ip-addresses configured to be private ip-addresses (the ones on physical ethernet). Once everything is puppetized, just bring down the br-ex on the compute nodes as its of no use.

Why we need br-ex on compute nodes:-
br-ex is used for flat and VLAN networks. We are using VXLAN and hence, it should not affect us. I am waiting for the tempest results to see what do they tell us about this configuration change.

This change is only for the compute nodes. If we keep it as it is, then openvswitch-agent fails on the compute nodes.